CONTRIBUTING.md

```diff
@@ -7,20 +7,20 @@ is to use Codespaces if you have access to it. If you do not, simply cloning the
 
 #### Steps
 
--   Cick the green dropdown labeled code, there should be two tabs: local and codespaces.
+-   Click the green dropdown labeled code, there should be two tabs: local and codespaces.
 -   Click on codespaces.
 -   If this is your first time, then create a new codespace. It will open a new browser tab and build the docker container for it - there will be a button to open the environment in VSCode if you'd prefer to run it that way
 -   You should see a screen that says `Setting up your codespace` As soon as that's done, you should see a VSCode like UI with files on the left.
 
-Once codespaces is up and runnng make sure `jq` is installed and then:
+Once codespaces is up and running make sure `jq` is installed and then:
```

Comment:
What is `jq`? I was trying to build locally but go stuck here.

Reply:
https://stedolan.github.io/jq/